shared/chat/inbox/index.desktop.tsx

```diff
@@ -61,7 +59,7 @@ class Inbox extends React.Component<T.Props, State> {
     // list changed
     if (
       this.props.rows.length !== prevProps.rows.length ||
-      !I.is(this.props.unreadIndices, prevProps.unreadIndices)
+      !shallowEqual(this.props.unreadIndices, prevProps.unreadIndices)
```

Comment:
If you receive a message that should make the "Unread messages" banner show, it doesn't until you scroll a bit. We need to `this.calculateShowUnreadShortcut` here. It also seems like these two conditions should independently cause `calculateShowFloating` and `calculateShowUnreadShortcut` (i.e. can be separate if statements), but I'm not sure about that.

Reply:
fixed, yeah it can be two different ifs